integration-tests/integration-test-script/utils.py

```diff
@@ -361,10 +361,10 @@ def fuzzy_traffic_match(numA, numB):
     # signs must not match
     if numA > 0 and numB > 0 or numA < 0 and numB < 0:
         return False
-    if numA > 0:
+    if numA >= 0:
```

Comment:
rebase to pull in this change from the already merged commit

Reply:
fixed